crates/autopilot/src/infra/persistence/mod.rs

```diff
@@ -300,10 +293,50 @@ impl Persistence {
             id: auction_id,
             orders,
             prices,
-            deadline,
             surplus_capturing_jit_order_owners,
         })
     }
+
+    /// Get competition winner.
+    pub async fn get_winner(
```

Comment:
Can we call this `get_winning_bid` and return a `domain::competition::Bid` (or even better reconstruct a `domain::competition::Solution`)?

Reply:
ended with `domain::competition::Solution`